libafl_qemu/libafl_qemu_build/src/build.rs

```diff
@@ -418,6 +422,29 @@ pub fn build(
     if !libafl_qemu_no_build {
         let mut build_cmd = build_qemu(&cc_compiler, &cpp_compiler, &libafl_qemu_build_dir, jobs);
 
+        // Also let cargo know when to rerun the build for custom qemu directories
+        if libafl_qemu_dir_from_env {
+            let build_files = Command::new("git")
```

Comment:
This is a cool hack, but it means everybody needs to have git installed, should we have a fallback for if that's not the case?

Reply:
We already assume git is installed btw, I think it's fine

Reply:
Thanks!
I think that is not a problem, since the lines above use git already for cloning the repo & also we have the `build_dep_check(&["git", "make"]);` check earlier in this file.